README.md

```diff
@@ -12,6 +12,7 @@ open index.html in your prefered browser and you are good to go.
 To get a local copy up and running follow these simple example steps.
 
 write git clone https://github.com/Ashe546/Portfolio.git on your terminal
+Go to https://ashe546.github.io/Portfolio/ to view the project.
 
```

Comment:
- [ ] [Optional] It will be better professionally if you add a section title before your deployment link.
you can do it like that:
```
## Deployment
This is the live deployment [link](https://ashe546.github.io/Portfolio/) of my project
```
and it will look like that:

## Deployment
This is the live deployment [link](https://ashe546.github.io/Portfolio/) of my project